pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -175,7 +175,7 @@ public synchronized void removeConsumer(Consumer consumer) throws BrokerServiceE
                     log.info("[{}] All consumers removed. Subscription is disconnected", name);
                     closeFuture.complete(null);
                 }
-                totalAvailablePermits = 0;
+                TOTAL_AVAILABLE_PERMITS_UPDATER.set(this, 0);
```

Comment:
This would be equivalent to `totalAvailablePermits = 0`

Reply:
true. I tried to make it consistent with the style used in NonPersistentDispatcherMultipleConsumers: https://github.com/apache/pulsar/blob/31f831574c3a65bae0e76801facaf2deb0b17fbb/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentDispatcherMultipleConsumers.java#L104